bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/VoiceManager.java

```diff
@@ -280,6 +280,29 @@ void listenAndAnswer(@Nullable STTService stt, @Nullable TTSService tts, @Nullab
      */
     Collection<KSService> getKSs();
 
+    /**
+     * Retrieves a HumanLanguageInterpreter collection.
+     * If no services are available returns null.
+     * 
+     * @param idList Comma separated list of HLI service ids to use or null
+     * @return a Collection<HumanLanguageInterpreter> or null, if no services are available or if some defaults are
+     *         configured, but no
+     *         according services are found
+     */
+    @Nullable
+    Collection<HumanLanguageInterpreter> getHLIByIds(String idList);
+
+    /**
+     * Retrieves a HumanLanguageInterpreter collection.
+     * If a default names are configured and the services are available, those are returned. Otherwise, the first
+     * available service is returned.
+     * 
+     * @return a Collection<HumanLanguageInterpreter> or null, if no services are available or if some defaults are
+     *         configured, but no according services are found.
+     */
+    @Nullable
+    Collection<HumanLanguageInterpreter> getDefaultHLIs();
```

Comment:
I really think we should keep a unique default interpreter.
So I would suggest to remove this method.

Reply:
But if we remove this it will be no possible to enable multi selection on the ui, I was planning to open a followup issue on the ui to ask for this functionally.
I think it makes sense, is something common that you want to use the same set of interpreters in all the places, if we remove this we are forcing the user to pass the interpreter ids in all places.
Allowing a list of interpreters on the call but not allowing it to be configured as default seems a little awkward for me.